client/src/components/modals/mras-search-info.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-dialog v-model="showModal" id="mras-search-info-modal" width="40rem" persistent>
+    <v-card class="pa-9">
+      <v-row no-gutters>
+        <v-col cols="11">
+          <v-card-title>
+            Corporate Number Not Found
+          </v-card-title>
+        </v-col>
+        <v-col cols="1">
+          <v-icon md color="primary" @click="clearAndClose()">mdi-close</v-icon>
+        </v-col>
+      </v-row>
+      <v-card-text class="copy-normal pb-3">
+        <p>Corporate Number assigned by Home Jurisdiction: <span class="font-weight-bold">{{ corpSearch }}</span><br>
+          Home jurisdiction: <span class="font-weight-bold">{{ jurisdictionText }}</span></p>
+        <p>{{ resultConfig[resultCode].desc  }}</p>
+        <p>{{ resultConfig[resultCode].action }}</p>
+      </v-card-text>
+      <NameInput id="name-input-component"
+                 class="mb-n7 pa-0"/>
+      <v-card-actions class="justify-center">
+        <div class="mt-1 mb-1 text-center">
+          <v-btn class="search-btn"
+                 @click="handleSubmit()">{{ showSearch ? 'Search' : 'Close' }}</v-btn>
+        </div>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import newReqModule from '@/store/new-request-module'
+import { Component, Vue } from 'vue-property-decorator'
+import NameInput from '@/components/new-request/name-input.vue'
+
+@Component({
+  components: { NameInput }
+})
+export default class MrasSearchInfoModal extends Vue {
+  private resultConfig = {
+    400: {
```

Comment:
Maybe we should import that HTTP codes package like we do in Filings UI?

Reply:
For sure, really good idea!

Reply:
https://www.npmjs.com/package/http-status-codes

Reply:
thanks!

Reply:
This was a great idea BTW. Such a simple thing but so much more clear. 💯